libclc/CMakeLists.txt

```
@@ -430,6 +412,7 @@ foreach( t ${LIBCLC_TARGETS_TO_BUILD} )
       # Error on undefined macros
       -Werror=undef
       -fdiscard-value-names
+      -Xclang -fdenormal-fp-math=dynamic
```

```suggestion
      -fdenormal-fp-math=dynamic
```

You can drop the `-Xclang` `-fdenormal-fp-math` is a driver option. (https://clang.llvm.org/docs/ClangCommandLineReference.html#cmdoption-clang-fdenormal-fp-math)